app/lib/efile/nc/d400_calculator.rb

```diff
@@ -96,7 +96,7 @@ def calculate_line_11
       def calculate_line_12a
         # Add Lines 9, 10b, and 11
         # line 9 DeductionsFromFAGI is blank
-        line_or_zero(:NCD400_LINE_10B) + line_or_zero(:NCD400_LINE_11)
+        calculate_line_9 + line_or_zero(:NCD400_LINE_10B) + line_or_zero(:NCD400_LINE_11)
```

Comment:
i think we want to use `line_or_zero` here!

Reply:
Sounds good, updated